controllers/configurationpolicy_controller.go

```diff
@@ -898,6 +898,8 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc *policyv1.Conf
 // validateConfigPolicy returns an error and increments the "invalid-template" error counter metric
 // if the configuration is invalid.
 func (r *ConfigurationPolicyReconciler) validateConfigPolicy(plc *policyv1.ConfigurationPolicy) error {
+	log := log.WithValues("policy", plc.GetName())
```

Comment:
These will help; my bad for missing that I took away some context they had.

At some point, thanks to the event-driven changes, we could consider passing the reconcile `ctx` to (some) of these functions - that has a logger which OperatorPolicy uses, and I feel like it's kinda nice. We might want to figure out if we can customize it if it has too many default key+values.

Reply:
Good idea @JustinKuli! If you're motivated, feel free to make that PR. I'll gladly review it.